kuma/core/jobs.py

```diff
@@ -18,3 +18,17 @@ def hash(self, value):
         if isinstance(value, tuple):
             value = tuple(to_bytestring(v) for v in value)
         return hashlib.md5(six.b(':').join(value)).hexdigest()
+
+
+class IPBanJob(KumaJob):
+    lifetime = 60 * 60 * 3
+    refresh_timeout = 60
+
+    def fetch(self, ip):
+        from .models import IPBan
+        if IPBan.objects.active(ip=ip).exists():
+            return "0/s"
+        return "60/m"
+
+    def empty(self):
+        return []
```

Comment:
I don't think it should return a list here since you're expecting a string in fetch. the docstring says:

        Return the appropriate value for a cache MISS (and when we defer the
        repopulation of the cache)

So probably something like `"60/m"` as well?